docs-v2/reference/scripts.mdx

```diff
@@ -56,37 +56,343 @@ Read more about [integration scripts](/understand/concepts/scripts) to understan
+        status: 200, // the HTTP status code
+        headers: {}, // the HTTP headers
+        config: {}, // the config provided for the request
+        request: {} // the request that generated this response
+    }
+```
+
+<Tip>
+    The response object is an [Axios response](https://axios-http.com/docs/res_schema).
+</Tip>
+</Expandable>
+
+### PUT requests
+
+Makes an HTTP PUT request inside an integration script.
+
+```js
+const res = await nango.put({
+    endpoint: '/endpoint',
+    baseUrlOverride: 'https://base-url.com'
+});
+```
+
+**Parameters**
+
+<Expandable>
+    <ResponseField name="config" type="object" required>
+        <Expandable title="config" defaultOpen>
+            <ResponseField name="endpoint" type="string" required>
+                The endpoint of the request.
+            </ResponseField>
+            <ResponseField name="headers" type="Record<string, string>">
+                The headers of the request.
+            </ResponseField>
+            <ResponseField name="params" type="Record<string, string | number>">
+                The query parameters of the request.
+            </ResponseField>
+            <ResponseField name="data" type="unkown">
+                The body of the request.
+            </ResponseField>
+            <ResponseField name="retries" type="number">
+                The number of retries in case of failure (with exponential back-off). Optional, default 0.
+            </ResponseField>
+            <ResponseField name="retryOn" type="number[]">
+                Array of additional status codes to retry a request in addition to the 5xx, 429, ECONNRESET, ETIMEDOUT, and ECONNABORTED
+            </ResponseField>
+            <ResponseField name="baseUrlOverride" type="string">
+                The API base URL. Can be ommitted if the base URL is configured for this API in the [providers.yaml](https://nango.dev/providers.yaml).
+            </ResponseField>
+            <ResponseField name="decompress" type="boolean">
+                Override the decompress option when making requests. Optional, defaults to false
+            </ResponseField>
+            <ResponseField name="responseType" type="'arraybuffer' | 'blob' | 'document' | 'json' | 'text' | 'stream'">
+                The type of the response. 
+            </ResponseField>
+        </Expandable>
+    </ResponseField>
+</Expandable>
+
+**Response**
+
+<Expandable>
+```json
+    {
+        data: {}, // the response provided by the server
+        status: 200, // the HTTP status code
+        headers: {}, // the HTTP headers
+        config: {}, // the config provided for the request
+        request: {} // the request that generated this response
+    }
+```
+
+<Tip>
+    The response object is an [Axios response](https://axios-http.com/docs/res_schema).
+</Tip>
+</Expandable>
+
+### PATCH requests
+
+Makes an HTTP PATCH request inside an integration script.
+
+Triggers an action for a connection.
```

Comment:
I'm not sure I get this sentence in this context

Reply:
Removed